src/components/StatelessTransaction.js

```diff
@@ -84,6 +113,12 @@ const StatelessTransaction = props => {
       </InnerLabel>
     </Input>
 
+  const maxTxnPriceInEth = (
+    <div className="mt-8">
+      Max transaction price: <span className="text-700">{(gasPrice * gasLimit) / 1000000000} eth</span>
```

Comment:
"Transaction *fee*" is canonical terminology here. I recommend changing that in the variable name also, just to stay consistent.

Reply:
I'd also give this a slightly smaller top margin, `8` is a bit much. `2` might look best to keep it nice and tied to the slider.

Reply:
Moved max transaction fee to the top right of the slider; what do you think?